app-admin/syslog-ng/syslog-ng-3.15.1.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit autotools python-single-r1 eutils multilib systemd versionator
+
+MY_PV=${PV/_/}
+MY_PV_MM=$(get_version_component_range 1-2)
+DESCRIPTION="syslog replacement with advanced filtering features"
+HOMEPAGE="https://syslog-ng.com/open-source-log-management"
+SRC_URI="https://github.com/balabit/syslog-ng/releases/download/${P}/${P}.tar.gz"
+
+LICENSE="GPL-2+ LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="amqp caps dbi geoip http ipv6 json libressl mongodb pacct python redis smtp spoof-source systemd tcpd"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+RESTRICT="test"
+
+RDEPEND="
+	amqp? ( >=net-libs/rabbitmq-c-0.8.0 )
+	caps? ( sys-libs/libcap )
+	dbi? ( >=dev-db/libdbi-0.9.0 )
+	geoip? ( >=dev-libs/geoip-1.5.0 )
+	http? ( net-misc/curl )
+	json? ( >=dev-libs/json-c-0.9:= )
+	mongodb? ( >=dev-libs/mongo-c-driver-1.2.0 )
+	python? ( ${PYTHON_DEPS} )
+	redis? ( >=dev-libs/hiredis-0.11.0 )
+	smtp? ( net-libs/libesmtp )
+	spoof-source? ( net-libs/libnet:1.1 )
+	systemd? ( sys-apps/systemd )
+	tcpd? ( >=sys-apps/tcp-wrappers-7.6 )
+	>=dev-libs/ivykis-0.36.1
+	>=dev-libs/libpcre-6.1
+	!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	!dev-libs/eventlog
+	>=dev-libs/glib-2.10.1:2"
+DEPEND="${RDEPEND}
+	sys-devel/flex
+	virtual/pkgconfig"
+
+DOCS=( AUTHORS NEWS.md CONTRIBUTING.md contrib/syslog-ng.conf.{HP-UX,RedHat,SunOS,doc}
+	contrib/syslog2ng "${FILESDIR}/${MY_PV_MM}/syslog-ng.conf.gentoo.hardened"
+	"${T}/syslog-ng.logrotate.hardened" "${FILESDIR}/README.hardened" )
+
+S=${WORKDIR}/${PN}-${MY_PV}
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	local f
+
+	use python && python_fix_shebang .
+
+	# remove bundled libs
+	rm -r lib/ivykis modules/afmongodb/mongo-c-driver modules/afamqp/rabbitmq-c || die
+
+	# drop scl modules requiring json
+	if use !json; then
+		sed -i -r '/cim|elasticsearch|ewmm|graylog2|loggly|logmatic/d' scl/Makefile.am || die
+	fi
+
+	# use gentoo default path
+	if use systemd; then
+		sed -e 's@/etc/syslog-ng.conf@/etc/syslog-ng/syslog-ng.conf@g;s@/var/run@/run@g' \
+			-i contrib/systemd/syslog-ng@default || die
+	fi
+
+	for f in "${FILESDIR}"/*logrotate*.in ; do
```

Comment:
Do not glob in FILESDIR. This will cause trouble if future version adds extra files.